web/views.py

```
@@ -6469,6 +6473,127 @@ def all_study_groups(request):
     )
 
 
+# file review views
+
+
+@login_required
+def upload_pdf_submission(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `upload_pdf_submission`

(ANN201)